React 17 support

react-textarea-autosize  6.1.0  does not support react 17, is there any reason for not using a newer version?

`
npm ERR! Could not resolve dependency:
npm ERR! peer react@">=0.14.0 <17.0.0" from react-textarea-autosize@6.1.0
npm ERR! node_modules/react-json-view/node_modules/react-textarea-autosize
npm ERR!   react-textarea-autosize@"^6.1.0" from react-json-view@1.21.1
npm ERR!   node_modules/react-json-view
npm ERR!     react-json-view@"^1.21.1" from the root project
`

--- a/package.json
+++ b/package.json
@@ -1,13 +1,13 @@
 {
     "name": "react-json-view",
     "description": "Interactive react component for displaying javascript arrays and JSON objects.",
-    "version": "1.21.1",
+    "version": "1.21.2",
     "main": "dist/main.js",
     "dependencies": {
         "flux": "^4.0.1",
         "react-base16-styling": "^0.6.0",
         "react-lifecycles-compat": "^3.0.4",
-        "react-textarea-autosize": "^6.1.0"
+        "react-textarea-autosize": "^8.3.2"
     },
     "devDependencies": {
         "@babel/eslint-parser": "^7.12.1",

--- a/src/js/components/VariableEditor.js
+++ b/src/js/components/VariableEditor.js
@@ -293,6 +293,7 @@ class VariableEditor extends React.PureComponent {
                         e.stopPropagation();
                     }}
                     placeholder="update this value"
+                    minRows={2}
                     {...Theme(theme, 'edit-input')}
                 />
                 <div {...Theme(theme, 'edit-icon-container')}>

--- a/src/js/themes/getStyle.js
+++ b/src/js/themes/getStyle.js
@@ -292,7 +292,6 @@ const getDefaultThemeStyling = theme => {
         },
         'edit-input': {
             display: 'inline-block',
-            minHeight: constants.editInputHeight,
             minWidth: constants.editInputMinWidth,
             borderRadius: constants.editInputBorderRadius,
             backgroundColor: colors.editVariable.background,

--- a/src/js/themes/styleConstants.js
+++ b/src/js/themes/styleConstants.js
@@ -69,7 +69,6 @@ export default {
 
     tooltipPadding: '4px',
 
-    editInputHeight: '25px',
     editInputMinWidth: '130px',
     editInputBorderRadius: '2px',
     editInputPadding: '5px',

